tests/phpunit/helpers/class-wc-pre-orders-product.php

```
@@ -30,10 +30,20 @@ public static function set_is_pre_order_charged_upon_release( bool $result ): vo
 		return self::$product_is_charged_upon_release_result;
 	}
+
+	/**
+	 * Stub: determine if a product is charged upfront (pre-order).
+	 *
+	 * @param WC_Product|object $product The product to check.
+	 * @return bool
+	 */
+	public static function product_is_charged_upfront( $product ): bool {
+		return false;
+	}
```

PHPMD (2.15.0)

[error] The method product_is_charged_upfront is not named in camelCase. (undefined)

(CamelCaseMethodName)